media_driver/linux/common/ddi/media_libva_caps.cpp

```diff
@@ -720,7 +720,13 @@ VAStatus MediaLibvaCaps::CreateEncAttributes(
     attrib.type = VAConfigAttribEncMaxRefFrames;
     if (entrypoint == VAEntrypointEncSliceLP)
     {
+        //VDEnc Low delay P
```

Comment:
This should be P frame, right?

Reply:
yes, the number of reference frame in L1 is 0.